src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an example in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  sourceString
+    .split(';')
+    .map((string) => string.trim())
+    .filter((string) => string)
+    .forEach((string) => {
+      const parts = string.split(':');
+      const [ part1, part2 ] = parts;
+
+      stylesObject[part1.trim()] = part2.trim();
+    });
```

Comment:
Chaining is a good thing, but I recommend rewriting the code so that it is more readable. 
Also, try to use one array method which can return an object.